examples/hello_world/client/nodejs/index.js

```diff
@@ -53,31 +46,34 @@ async function main() {
     return metaData;
   }
 
-  const helloWorldProto = grpc.loadPackageDefinition(helloWorldDefinition).oak
-    .examples.hello_world;
+  const helloWorldService = protos.lookupService(
+    'oak.examples.hello_world.HelloWorld'
+  );
   const credentials = grpc.credentials.createSsl(fs.readFileSync(CERT_PATH));
 
-  const client = new helloWorldProto.HelloWorld('localhost:8080', credentials);
+  const grpcClient = new grpc.Client('localhost:8080', credentials);
 
-  // For documentation on client calls see the `@grpc/grpc-js` documentation:
-  // https://grpc.github.io/grpc/node/grpc.Client.html#~CallProperties
-  client.sayHello(
-    // The arguments passed to the gRPC service. Corresponds to the
-    // `HelloRequest` message type in hello_world.proto file.
-    { greeting: 'Node.js' },
-    // The metadata for this gRPC call.
-    getGrpcMetadata(),
-    // Callback invoked with the response.
-    (error, response) => {
-      if (error) {
-        console.error(error);
-        process.exit(1);
-      } else {
-        console.log(response.reply);
-        process.exit(0);
-      }
+  const HelloWorld = helloWorldService.create(
```

Comment:
(nit) is the convention to start identifiers with upper or lower case letter?

Reply:
Generally only constructors should be capitalized, this was a bit inconsistent indeed, thx for catching. :)